source/includes/administration/_billing_settings.md

```diff
@@ -68,21 +68,53 @@ curl "https://cloudmc_endpoint/api/v2/reseller/settings/billing/f7ad28a8-1227-44
 
 ```json
 {
-  "data": {
-    "organization": {
-      "id": "10572c3d-16e5-450f-8af8-a01e50dc52d4"
-    },
-    "id": "f7ad28a8-1227-44de-9785-6dbd556f3bda",
-    "version": 1,
-    "daysBeforeAutoDraft": 3,
-    "daysBeforeAutoApproval": 3,
-    "daysBeforeAutoPayment": 5,
-    "bccEmails": ["finance.support@company.com", "monitoring@company.com"],
-    "daysBeforeCardWarnings": [30],
-    "customerInformation": ["accountId"],
-    "address": ["Address line 1", "Address line 2", "etc."],
-    "termsAndConditions": "My terms and conditions..."
-  }
+   "data":{
+      "daysBeforeAutoDraft":5,
+      "minimumDaysInvoicePeriod":15,
+      "bccEmails":[
+         "email@cloudops.com"
+      ],
+      "address":{
+         "country":"USA",
+         "province":"Georgia",
+         "city":"Atlanta",
+         "postalCode":"30328",
+         "businessName":"Org",
+         "lineOne":"123 Example Street"
+      },
+      "daysBeforeAutoApproval":3,
+      "version":93,
+      "defaultPricingPackage":{
+         "id":"8358f8d6-1f2d-4bb9-bb0a-f8ce51dfa19c"
+      },
+      "daysBeforeCardWarnings":[
+         24
+      ],
+      "termsAndConditions":"Terms",
+      "defaultCurrency":"USD",
+      "organization":{
+         "name":"Org",
+         "customFieldDefinitions":[
+            {
+               "field":"account_number",
+               "id":"31b7350a-3c41-4d64-b58b-c7bfbc45abbb",
+               "nameTranslations":{
+                  "en":"Account Number"
+               },
+               "descriptionTranslations":{
+                  "en":"Account number"
+               }
+            }
+          ],
```

Comment:
is this needed in the response?

Reply:
I felt that it would be nice to have in context, however I should update it to state that it only returns definitions if there are any